.github/workflows/build-sealos-cluster-image.yml

```diff
@@ -97,6 +97,21 @@ jobs:
           sudo buildah manifest add $IMAGE:$VERSION docker://$IMAGE:$VERSION-arm64
           sudo buildah manifest push --all $IMAGE:$VERSION docker://$IMAGE:$VERSION
 
+      - name: Renew issue and Sync Images
+        uses: labring/gh-rebot@v0.0.6
+        if: ${{ github.repository_owner == labring }}
+        with:
+          version: v0.0.8-rc1
+        env:
+          GH_TOKEN: "${{ secrets.GH_PAT }}"
+          SEALOS_TYPE: "issue_renew"
+          SEALOS_ISSUE_TITLE: "【DaylyReport】 Auto build for laf"
+          SEALOS_ISSUE_BODYFILE: "build/README.md"
+          SEALOS_ISSUE_LABEL: "dayly-report"
```

Comment:
dayly -> daily

Reply:
https://github.com/labring-actions/cluster-image/issues/376 

Temporarily now? If you jump to sealos, the item will probably have to be changed